### PR TITLE
feature(Widgets): add a widget to display a scale

### PR DESCRIPTION
**Pull request by mgermerie**
## Description

Add a widget to display a scale

### CODE REVIEW

File: examples/misc_orthographic_camera.html [hunk 1]
```diff
@@ -75,22 +75,18 @@
...
             // Updates scale when needed :
             view.addEventListener(itowns.VIEW_EVENTS.INITIALIZED, function () {
                 // eslint-disable-next-line no-console
                 console.info('View initialized');
-                updateScaleWidget();
+                scale.update();
             });
             view.addEventListener(itowns.PLANAR_CONTROL_EVENT.MOVED, function () {
-                updateScaleWidget();
+                scale.update();
```
Comment (gchoqueux) [inline examples/misc_orthographic_camera.html]: could you move in scale widget?

File: examples/widgets_scale.html [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                translate: { x: -120 },
+            });
+
+            // Update scale when needed.
+            view.addEventListener(itowns.GLOBE_VIEW_EVENTS.GLOBE_INITIALIZED, function () {
+                scale.update();
+            });
+            view.controls.addEventListener(itowns.CONTROL_EVENTS.RANGE_CHANGED, () => {
+                scale.update();
+            });
```
Comment (gchoqueux) [inline examples/widgets_scale.html]: could you move in scale widget?

File: examples/misc_orthographic_camera.html (hunk 1 above)
Comment (mgermerie) [reply examples/misc_orthographic_camera.html]: I switched this to Scale.

File: examples/widgets_scale.html (hunk 2 above)
Comment (mgermerie) [reply examples/widgets_scale.html]: I switched this to Scale.

**Review by gchoqueux — APPROVED** [review]

**Pull request merged**